### PR TITLE
Add missing test cases

### PR DESCRIPTION
**Pull request by levanhieuhust**
**What is the purpose of the change**

- Update test case for **keeper** and **types** of Pylons
- Write all test cases with table-driven test for **keeper** and **types** of Pylons

**Testing and Verifying**

- This change is a trivial rework/code cleanup without any test coverage

**Documentation and Release Note**

- Does this pull request introduces a new feature or user-facing behaviour changes? no
- How is the feature or change documented? not applicable

### CODE REVIEW

File: x/pylons/types/payment_processor_test.go [hunk 1]
```diff
@@ -72,6 +72,27 @@ func TestPaymentProcessor_ValidatePaymentInfo(t *testing.T) {
 			},
 			wantErr: false,
 		},
+		{
+			name: "InvalidPubkey",
+			fields: fields{
+				CoinDenom:            "ustripeusd",
+				PubKey:               "INVALID_PUBKEY",
+				ProcessorPercentange: DefaultProcessorPercentage,
```
Comment (mustafapylons) [inline x/pylons/types/payment_processor_test.go:80]: ProcessorPercentage is correct spelling.

File: x/pylons/types/payment_processor_test.go [hunk 2]
```diff
@@ -93,6 +114,27 @@ func TestPaymentProcessor_ValidatePaymentInfo(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
+			name: "InvalidName",
+			fields: fields{
+				CoinDenom:            "ustripeusd",
+				PubKey:               base64.StdEncoding.EncodeToString(privKey.PubKey().Bytes()),
+				ProcessorPercentange: DefaultProcessorPercentage,
```
Comment (mustafapylons) [inline x/pylons/types/payment_processor_test.go:122]: ProcessorPercentage is correct spelling.

File: x/pylons/types/payment_processor_test.go [hunk 3]
```diff
@@ -148,6 +190,26 @@ func TestPaymentProcessor_ValidateRedeemInfo(t *testing.T) {
 			},
 			wantErr: false,
 		},
+		{
+			name: "InvalidPubkey",
+			fields: fields{
+				CoinDenom:            "ustripeusd",
+				PubKey:               "INVALID_PUBKEY",
+				ProcessorPercentange: DefaultProcessorPercentage,
```
Comment (mustafapylons) [inline x/pylons/types/payment_processor_test.go:198]: ProcessorPercentage is correct spelling.

File: x/pylons/types/payment_processor_test.go [hunk 4]
```diff
@@ -168,6 +230,26 @@ func TestPaymentProcessor_ValidateRedeemInfo(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
+			name: "InvalidName",
+			fields: fields{
+				CoinDenom:            "ustripeusd",
+				PubKey:               base64.StdEncoding.EncodeToString(privKey.PubKey().Bytes()),
+				ProcessorPercentange: DefaultProcessorPercentage,
```
Comment (mustafapylons) [inline x/pylons/types/payment_processor_test.go:238]: ProcessorPercentage is correct spelling.

File: x/pylons/keeper/msg_server_account_test.go [hunk 5]
```diff
@@ -97,6 +102,16 @@ func (suite *IntegrationTestSuite) TestUpdateAccount() {
 			request: &types.MsgUpdateAccount{Creator: addr[0], Username: "testUserUpdated"},
 			err:     types.ErrDuplicateUsername,
 		},
+		{
+			desc:    "Account not created",
+			request: &types.MsgUpdateAccount{Creator: addr[1], Username: "testUserUpdated1"},
+			err:     sdkerrors.ErrInvalidRequest,
+		},
+		{
+			desc:    "Invalid payfee",
```
Comment (mustafapylons) [inline x/pylons/keeper/msg_server_account_test.go:111]: Description probably would read "Invalid fee"

File: x/pylons/types/genesis_test.go [hunk 6]
```diff
@@ -1,17 +1,146 @@
...
+			req: &GenesisState{
+				RedeemInfoList: []RedeemInfo{
+					{Id: "test"},
+					{Id: "test"},
+				},
+			},
+			err: fmt.Errorf("duplicated index for redeemInfo"),
+		},
+		{
+			desc: "Duplicate index of Redeminqo",
```
Comment (mustafapylons) [inline x/pylons/types/genesis_test.go:35]: "redeeminfo" instead of "Redeminqo".

**Review by mustafapylons — COMMENTED** [review]
some typos to fix, otherwise it's good.

File: x/pylons/types/payment_processor_test.go (hunk 1 above)
Comment (levanhieuhust) [reply x/pylons/types/payment_processor_test.go]: Thank you very much! I have fix in commit https://github.com/Pylons-tech/pylons/pull/870/commits/8aa682a4948fce1f64f88c04f253cbb7e3b4e2f1

File: x/pylons/types/payment_processor_test.go (hunk 2 above)
Comment (levanhieuhust) [reply x/pylons/types/payment_processor_test.go]: Thank you very much! I have fix in commit https://github.com/Pylons-tech/pylons/pull/870/commits/8aa682a4948fce1f64f88c04f253cbb7e3b4e2f1

File: x/pylons/types/payment_processor_test.go (hunk 3 above)
Comment (levanhieuhust) [reply x/pylons/types/payment_processor_test.go]: Thank you very much! I have fix in commit https://github.com/Pylons-tech/pylons/pull/870/commits/8aa682a4948fce1f64f88c04f253cbb7e3b4e2f1

File: x/pylons/types/payment_processor_test.go (hunk 4 above)
Comment (levanhieuhust) [reply x/pylons/types/payment_processor_test.go]: Thank you very much! I have fix in commit https://github.com/Pylons-tech/pylons/pull/870/commits/8aa682a4948fce1f64f88c04f253cbb7e3b4e2f1

File: x/pylons/keeper/msg_server_account_test.go (hunk 5 above)
Comment (levanhieuhust) [reply x/pylons/keeper/msg_server_account_test.go]: Thank you very much! I have fix in commit https://github.com/Pylons-tech/pylons/pull/870/commits/8aa682a4948fce1f64f88c04f253cbb7e3b4e2f1

File: x/pylons/types/genesis_test.go (hunk 6 above)
Comment (levanhieuhust) [reply x/pylons/types/genesis_test.go]: Thank you very much! I have fix in commit https://github.com/Pylons-tech/pylons/pull/870/commits/8aa682a4948fce1f64f88c04f253cbb7e3b4e2f1

**Review by mustafapylons — APPROVED** [review]
Looks good to me now.

**Comment by levanhieuhust** [comment]
Hello @faddat , could you please review for this pr? Thank sir

**Comment by levanhieuhust** [comment]
Hello @faddat . Could you please check for this pull request. Thank Sir !

**Review by faddat — APPROVED** [review]

**Pull request merged**